iconservice/prep/data/prep_container.py

```diff
@@ -188,12 +191,17 @@ def get_by_address(self, address: 'Address') -> Optional['PRep']:
         """
         return self._prep_dict.get(address)
 
-    def get_preps(self, start_index: int, size: int) -> List['PRep']:
+    def get_preps(self, start_index: int, size: int, active_only: bool = False) -> List['PRep']:
         """Returns active P-Reps ranging from start_index to start_index + size - 1
 
         :return: P-Rep list
         """
-        return self._active_prep_list[start_index:start_index + size]
+        if active_only:
```

Comment:
logic is ok, but the meaning is opposite

Reply:
fix logic